support/dev/index_generator.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+/**
+ * generates an index.json file for use when running Titanium under xcode
+ */
+'use strict';
+
+const fs = require('fs-extra');
+const path = require('path');
+
+const dirToTraverse = process.argv[2];
+// TODO: gather all the js/json files under this dir and generate and index.json file in it
+
+const index = {};
+const destFile = path.join(dirToTraverse, 'index.json');
+const parentDir = path.dirname(dirToTraverse);
```

Comment:
<!--
  0 failure: 
  1 warning:  support/dev/index...
  
  
  DangerID: danger-id-default;
  File: support/dev/index_generator.js;
  Line: 15;
-->

- :warning: support/dev/index_generator.js line 15 – 'parentDir' is assigned a value but never used. (no-unused-vars)